cmd/dmverity-vhd/main.go

```diff
@@ -25,6 +26,7 @@ const (
 	imageFlag         = "image"
 	verboseFlag       = "verbose"
 	outputDirFlag     = "out-dir"
+	sourceFlag        = "local"
```

Comment:
Maybe `docker` would be a better name, since we are reading from a local Docker daemon specifically?

Reply:
I think that could be unclear since Docker can generally pull local and remote, although this is only using it for the local.

Reply:
The current naming is unclear because local could mean a local OCI repository, local containerd instance, local tar files, etc. None of which we are supporting here. So I'd prefer we are specific about what we're doing (which per my understanding, is reaching out to the Docker daemon to get layers from its store).